components/Navbar.tsx

```diff
@@ -1,86 +1,104 @@
-import { useWallet } from '@solana/wallet-adapter-react'
+import { FC, useState } from 'react'
+import Image from 'next/image'
+import { WalletMultiButton } from '@solana/wallet-adapter-react-ui'
+import { NavList, Breadcrumbs } from '@primer/react'
+import { useRouter } from 'next/router'
 import Link from 'next/link'
-import { FC } from 'react'
-
-interface INavbarElement {
-    title: string
-    url: string
-    tooltip?: string
-    disabled?: boolean
-}
+import { INavbarElement } from 'lib/interfaces'
 
 const Navbar: FC = () => {
-    const { publicKey } = useWallet()
-
+    const router = useRouter()
+    const [open, setOpen] = useState(false)
     const NAVBAR_ELEMENTS: INavbarElement[] = [
         {
-            title: 'Manage CMs',
+            title: 'Dashboard',
             url: '/',
```

Comment:
The path of `/` should be `Dashboard/Candy Machines`

Reply:
you mean the title right?